scripts/patch-release/patch-release.go

```diff
@@ -24,6 +24,8 @@ const (
 	branchPrefix      = "release-"
 )
 
+var releaseVersions = []string{"1.29.4", "1.28.4", "1.27.4"}
```

Comment:
This should be just the release branches:

```suggestion
var releaseBranches = []string{"release-1.29", "release-1.28", "release-1.27"}
```

Or you just use something like `releaseMinorVersions` which is then `1.29`, `1.28` and you automatically preprend `release-`.

Reply:
I think i would go with the releaseMinorVersions and prepend the "release-"